Makefile

```diff
@@ -60,7 +60,7 @@ ifeq ($(ENV_NAME), test)
 DOMAIN=test.ien.freshworks.club
 BASTION_INSTANCE_ID = $(BASTION_INSTANCE_ID_TEST)
 DB_HOST = $(DB_HOST_PROD_TEST)
-NEXT_PUBLIC_AUTH_URL=https://keycloak.freshworks.club/auth/realms/ien/protocol/openid-connect/certs
+NEXT_PUBLIC_AUTH_URL=https://keycloak.freshworks.club/auth
```

Comment:
Does this need too?
`NEXT_PUBLIC_AUTH_REALM=ien`

Reply:
You are correct, I will fix this